### PR TITLE
Add ogg_ignoretrack0 cvar to control default music playback

### PR DESCRIPTION
**Pull request by ewe2**
This patch is an alternate to #96 using a cvar. As the function logic currently stands, this will result in a harmless message that track 00 is not in the playlist when ogg_ignoretrack0 is set to 1. Some documentation of the cvar will be needed if accepted.

### CODE REVIEW

**Comment by ewe2** [comment]
Good point. Have updated the pull request.

**Comment by DanielGibson** [comment]
ogg_track0 is not very descriptive

**Comment by ewe2** [comment]
ogg_toggletrack0, ogg_ignoretrack0, ogg_keeploop, ogg_ignoregoal ? Pick something.

**Comment by DanielGibson** [comment]
ogg_ignoretrack0 sounds good to me, but in the end it's up to @Yamagi :)

**Comment by ewe2** [comment]
It's a bit wordy but at least it means something :D

**Comment by Yamagi** [comment]
I've merged this pull request manually after combining the commits into one.
